### PR TITLE
explorer: improve embedded PE detection

### LINKED ISSUE #1333: check_segment_for_pe in IDA extractor does unnecessary search
I think there is no need to push the next address into the todo list at https://github.com/mandiant/capa/blob/master/capa/features/extractors/ida/file.py#L64-L65, since we have already found all `mzx` at https://github.com/mandiant/capa/blob/master/capa/features/extractors/ida/file.py#L41-L44

### PR DESCRIPTION
**Pull request by mike-hunhoff**
closes #1333

### CODE REVIEW

**Review by williballenthin — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 - match: extend OS characteristic to match OS_ANY to all supported OSes #1324 @mike-hunhoff
 - explorer: fix exception when plugin loaded in IDA hosted under idat #1341 @mike-hunhoff
 - extractor: fix IDA and vivisect string and bytes features overlap and tests #1327 #1336 @xusheng6 
+- explorer: improve embedded PE detection #1344 @mike-hunhoff
```
Comment (mr-tz) [inline CHANGELOG.md:30]: there's also the explorer section just below for these

**Review by mr-tz — APPROVED** [review]
great, thanks, please merge or update the changelog entry first

File: CHANGELOG.md (hunk 1 above)
Comment (williballenthin) [reply CHANGELOG.md:30]: nit: maybe describing the improvement would be even more useful to an interested reader.

Comment (williballenthin) [reply CHANGELOG.md:30]:
```suggestion
- explorer: improve embedded PE detection performance and reduce FP potential #1344 @mike-hunhoff
```

**Review by williballenthin — APPROVED** [review]

**Pull request merged**